.github/workflows/docs-gen-and-push.yaml

```diff
@@ -125,7 +125,10 @@ jobs:
 
       # copy back older versions to the current docs
       - name: Copy everything from archive to docs
-        run: rsync -av docs/v* public
+        run: |
+          if ls docs/v* >/dev/null; then
+            rsync -av docs/v* public
+          fi
```

Comment:
Should we add an else branch as well, printing "No such folder"? will be easier to track in logs. (probably an overkill)